modules/react/_examples/stories/SidePanelWithNavigation.stories.mdx

```diff
@@ -0,0 +1,14 @@
+import {Meta, Story} from '@storybook/addon-docs';
+import {Basic} from './examples/SidePanelWithNavigation';
+
+<Meta title="Examples/Side Panel With Navigation" />
```

Comment:
There's another Side Panel example, I think we can consolidate them into a folder for `SidePanel/Basic` and `SidePanel/Overlay`